### PR TITLE
Enhancement added - Now copy Ayah feature includes Surah and Ayah number

### PR DESCRIPTION
**Pull request by imomer**
I have enhanced the copy feature so it can take the Surah ID and Ayah number when user copy an Ayah...

The return of it is like: `اهْدِنَا الصِّرَاطَ الْمُسْتَقِيمَ - 1:6`

### CODE REVIEW

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32828

**Comment by mmahalwy** [comment]
thanks @imomer ! I'd like to wait till the v3 branch is good to go though as it changes the naming of everything.

**Comment by imomer** [comment]
@mmahalwy Sure ... Thanks 🙂

**Comment by mmahalwy** [comment]
@imomer can you rebase so we can merge this :)

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32821

**Comment by imomer** [comment]
@mmahalwy Rebased :)

**Comment by sabeurthabti** [comment]
@imomer you have a conflict

**Comment by imomer** [comment]
@mmahalwy @sabeurthabti is src/components/Ayah/ is removed?

I'm seeing new component /Verse

**Comment by imomer** [comment]
I didn't know that Ayah component was replaced with Verse... Fixed it

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32822

File: .env [hunk 1]
```diff
@@ -1,6 +1,6 @@
 NODE_ENV=development
 PORT=8000
-API_URL=http://staging.quran.com:3000
+API_URL=http://quran.com
```
Comment (mmahalwy) [inline .env]: Yikes, can we make this staging again?

Comment (imomer) [inline .env]: Done

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32825

**Comment by ahmedre** [comment]
Deployed to: http://staging.quran.com:32831

**Comment by mmahalwy** [comment]
Thank you @imomer !

**Pull request merged**